### DISCUSSION TITLE
Import Topics from csv

### DISCUSSION THREAD

**Discussion by whvneo** [Zuehlke/poinz#217] [Ideas]
Hi, 

I really like your Planning Poker App. 
It woul dbe very helpful to be able to import a CSV file with TOPIC and URL. Or an API to use a script to create the topics to be discussed.

Regards
Andreas

**Comment by xeronimus** [comment]
Hi Andreas, @whvneo ,

thanks! I'm glad that Poinz is of use to you and your team! :)

Do you already know the csv import feature? https://github.com/Zuehlke/poinz/blob/master/docu/manual.md#importing-stories
A csv file that looks like this...

```
key,title,description
ISS-123,The Title, some Description with a url: https://my-super.jira.com/ISSUE_123
ISS-554,Another Story, With a description
```

...can contain URLs in the "description".   

You just drag 'n drop it to the story-backlog on the left side of the Poinz screen.


![image](https://user-images.githubusercontent.com/1777143/135084560-ab68e153-86f2-46b8-b4e3-ec0c21b86694.png)

**Reply by xeronimus** [reply]
@whvneo   does this "import" meet your needs or shall we start discussing a "scriptable" solution (via REST API) ?

let me know, if you need further advise..

**Comment by whvneo** [comment]
Hi @xeronimus , yes that fit's my requirement... Sorry, but I missed to read the manual :-)

Thanks for the hint

Regards 
Andreas

**Reply by xeronimus** [reply]
no worries :)  have fun!
